lib/class-wp-theme-json-gutenberg.php

```diff
@@ -3480,4 +3413,189 @@ public function set_spacing_sizes() {
 
 		_wp_array_set( $this->theme_json, array( 'settings', 'spacing', 'spacingSizes', 'default' ), $spacing_sizes );
 	}
+
+	/**
+	 * Generates the root selector for a block.
+	 *
+	 * @param object $block_type The block type.
+	 * @return string
+	 */
+	protected static function get_root_block_selector( $block_type ) {
```

Comment:
This function can be removed, can't it?

Reply:
Yep. It's no longer used after the switch to the global PHP function for determining selectors.

Thanks!

Reply:
✅ Done: 372c6f8bb9